appinventor/appengine/src/com/google/appinventor/client/actions/NewAction.java

```diff
@@ -0,0 +1,15 @@
+package com.google.appinventor.client.actions;
```

Comment:
1) Consider renaming this action to be "NewProjectAction"
2) Add copyright header